textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/lingpipeexample/LingpipeExample.java

```diff
@@ -0,0 +1,61 @@
+package edu.uci.ics.textdb.sandbox.lingpipeexample;
+
+import com.aliasi.chunk.Chunker;
+import com.aliasi.chunk.Chunking;
+
+import com.aliasi.util.AbstractExternalizable;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import com.aliasi.chunk.Chunk;
+ 
+
+public class LingpipeExample {
+
+    public static void main(String[] args) throws Exception {
+  
+        String model = "./lingpipe-resources/ne-en-bio-genetag.HmmChunker";
+        String dataFile = "./lingpipe-resources/abstract_100.txt";
```

Comment:
Move everything in `lingpipe-resources` to  `textdb/textdb-sandbox/src/main/resources`. Java convention is to put resources in a folder named `resources` in `main`.